src/site/content/en/handbook/markup-cards-collection/index.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: handbook
+title: Sample card layouts on the collection page
+date: 2019-06-26
+description: |
+  A sample usage of cards on the collection page for web.dev
+---
+
+The content and layout of cards are flexible, the content creator can change the content and adjust the design, and position of a card by using YAML properties.
+
+The collection of cards offers a grid-based layout system, with rows and columns. The content creator can adjust the position card by specifying the card layout, vertical and horizontal, to avoid unwanted gaps. However, the rendering of a grid-based layout could create some gaps if we do not determine the column and row.
```

Comment:
```suggestion
The collection of cards offers a grid-based layout system, with rows and columns. Adjust the position card by specifying the card layout, vertical and horizontal, to avoid unwanted gaps. 
```